### PR TITLE
Added jakarta.activation to pom file

### PR DESCRIPTION
**Pull request by reshmabidikar**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1,21 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!--
```
Comment (pierre) [inline pom.xml:2]: It looks like the header was reformatted -- could you undo this change?

**Review by pierre — APPROVED** [review]
Looks good! Just one minor comment.

**Review by sbrossie — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (reshmabidikar) [reply pom.xml:2]: Sorry about this, I did not notice. I've reverted it now.
